pkg/networkmanager/helpers.js

```diff
@@ -0,0 +1,43 @@
+import cockpit from "cockpit";
+
+export function getNetworkGraphMetrics() {
```

Comment:
There should be only one channel open for the textual usage samples, I'd say, like in the old code.  Now both NetworkPage and NetworkInterfacePage create their own,  (Ideally we should take the textual samples from the graph without any extra channel...)